spec/services/location_data_service_spec.rb

```diff
@@ -172,6 +172,17 @@
       LocationDataService.delete_existing_and_repopulate
       library_record = Locations::Library.find_by(code: 'annex')
     end
+
+    it "sets a static ID" do
+      LocationDataService.delete_existing_and_repopulate
```

Comment:
Can we run this twice in the test to validate idempotence? Or if we're sure it's already been done can we add a comment pointing to where it's done during setup or whatever?